src/index.html

```diff
@@ -5,8 +5,467 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="/" class="logo">
+        <img 
+          src="images/logo.png" 
+          alt="logo" 
+          class="logo__image"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a 
+              href="Apple" 
+              class="nav__link is-active"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a 
+              href="Samsung" 
+              class="nav__link"
+            >
+              Samsung
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a 
+              href="Smartphones" 
+              class="nav__link"
+            >
+              Smartphones
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a 
+              href="Laptops & computers" 
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              Laptops & computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a 
+              href="Gadgets" 
+              class="nav__link"
+            >
+              Gadgets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a 
+              href="Tablets" 
+              class="nav__link"
+            >
+              Tablets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a 
+              href="Photo" 
+              class="nav__link"
+            >
+              Photo
+            </a>
+          </li>
+          
+          <li class="nav__item">
+            <a 
+              href="Video" 
+              class="nav__link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <div class="menu">
+        <div class="card">
+          <img 
+            class="card__image"
+            data-qa="card" 
+            src="./images/imac.jpeg" 
+            alt="iMac">
+          <h2
+            class="card__title">
```

Comment:
Pay attention on closed bracket. Fix all places
```suggestion
          <img 
            class="card__image"
            data-qa="card" 
            src="./images/imac.jpeg" 
            alt="iMac"
          >
          <h2 class="card__title">
```